app/assets/stylesheets/styles/_header.scss

```diff
@@ -0,0 +1,7 @@
+nav {
+  background-color: #333;
+}
+
+.logo_image{
```

Comment:
Selector `logo_image` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space